docs/resources/Guild.md

```diff
@@ -389,6 +389,9 @@ Adds a user to the guild, provided you have a valid oauth2 access token for the
 > info
 > All parameters to this endpoint except for `access_token` are optional.
 
+> info
+> The Authorization header must be Bot authorization, with the token belonging to the application's bot.
```

Comment:
I think this could be:

"The Authorization header must be a Bot token, and that bot must be a member of the guild."

Reply:
The problem I have with this is the same reason I stated above

> I've seen someone use a bot not associated with the application before and get confused by that also. I'd like to make it non ambiguous. Any suggestions?

The "In the guild" part sounds reasonable though, but I'll wait to add it for now.